spyder/utils/conda.py

```diff
@@ -132,9 +141,15 @@ def get_list_conda_envs():
     if conda is None:
         return env_list
 
-    cmdstr = ' '.join([conda, 'env', 'list', '--json'])
+    if is_conda_based_app():
+        # Need root_prefix to remove Spyder's runtime environment
+        cmdstr = ' '.join([conda, 'info', '--json'])
+    else:
+        # micromamba info does not provide envs, so must keep env list
+        cmdstr = ' '.join([conda, 'env', 'list', '--json'])
+
     try:
-        out, __ = run_shell_command(cmdstr, env={}).communicate()
+        out, __ = run_shell_command(cmdstr).communicate()
```

Comment:
Why the `env` is not empty anymore here?

Reply:
That is because `conda.exe` needed the `USERPROFILE` environment variable to write to temporary file. However, this can reverted since we aren't going to use `conda.exe`.